src/test/java/org/jabref/logic/search/query/SearchQueryLuceneConversionTest.java

```diff
@@ -10,64 +12,55 @@
-                Arguments.of("( title:chocolate OR author:smith ) AND ( year:2024 )", "(title contains chocolate or author matches smith) AND (year = 2024)"),
-                Arguments.of("any:video AND year:1932", "video and year == 1932"),
-                Arguments.of("title:neighbou?r", "title =neighbou?r"),
-                Arguments.of("abstract:model\\{1,2\\}ing", "abstract = model{1,2}ing"),
-                Arguments.of("any:* AND -title:chocolate", "title != chocolate"),
-                Arguments.of("any:* AND -title:chocolate", "not title contains chocolate"),
-                // https://github.com/JabRef/jabref/issues/11654#issuecomment-2313178736
-                Arguments.of("( groups:\\:paywall AND -file:/.+/ ) AND -groups:\\/exclude", "groups=:paywall and file!=\"\" and groups!=/exclude"),
+                Arguments.of("content:\"two term\" annotations:\"two term\"", "\"two terms\""),
+                Arguments.of("content:\"two term\" annotations:\"two term\"", "any = \"two terms\""),
 
-                Arguments.of("( any:* AND -( groups:alpha ) ) AND ( any:* AND -( groups:beta ) )", "NOT(groups=alpha) AND NOT(groups=beta)"),
-                // not converted, because not working in JabRef 5.x
-                // Arguments.of("title:\"image processing\" OR keywords:\"image processing\"", "title|keywords = \"image processing\""),
+                Arguments.of("content:imag", "content = image"),
+                Arguments.of("annotations:imag", "annotations = image"),
+                Arguments.of("content:\"imag process\"", "content = \"image processing\""),
+                Arguments.of("+content:imag +annotations:process", "content = image AND annotations = processing"),
+                Arguments.of("+(content:imag annotations:process) +(content:term annotations:term)", "(content = image OR annotations = processing) AND term"),
+                Arguments.of("(content:on annotations:on) (+(content:two annotations:two) +(content:three annotations:three))", "one OR (two AND three)"),
 
-                // not converted, because wrong syntax for JabRef 5.x
-                // Arguments.of("( author:miller OR title:\"image processing\" OR keywords:\"image processing\" ) AND NOT author:brown AND NOT author:blue", "(author = miller or title|keywords = \"image processing\") and not author = brown and != author = blue"),
+                Arguments.of("(-content:term) (-annotations:term)", "any != term"),
+                Arguments.of("(-content:term) (-annotations:term)", "any !== term"),
+                Arguments.of("(-content:term) (-annotations:term)", "any !=! term"),
+                Arguments.of("(-content:\"two term\") (-annotations:\"two term\")", "any != \"two terms\""),
+                Arguments.of("+(-content:imag) +(-annotations:process)", "content != image AND annotations != processing"),
 
-                // String with a space
```

Comment:
You can keep this comment, because it is not that easy to read the intention of these group of tests.